test/src/helpers.h

```diff
@@ -76,6 +76,19 @@ extern std::mutex catch2_macro_mutex;
 #define REQUIRE_TILEDB_OK(a) \
   { REQUIRE(a == TILEDB_OK); }
 
+// A variant of the CHECK macro for checking a Status object is okay.
+#define CHECK_TILEDB_STATUS_OK(status) \
+  {                                    \
+    INFO(status.to_string());          \
```

Comment:
This being the INFO statement or the whole macro? I intentionally left the info statement in, since it is useful to see the status message when the check fails.

Reply:
How about making this conditional, print on failure only? I really like the idea (it's annoying to see test failures in the C API with no error handling) but printing unconditionally seems like it will increase verbosity quite a bit.